client/state/selectors/user-has-any-atomic-sites.js

```diff
@@ -17,6 +12,7 @@ import isAtomicSite from 'calypso/state/selectors/is-site-automated-transfer';
  * @returns {boolean}
  */
 export default createSelector(
-	( state ) => some( getSitesItems( state ), ( site ) => isAtomicSite( state, site.ID ) ),
+	( state ) =>
+		Object.values( getSitesItems( state ) ).some( ( site ) => isAtomicSite( state, site.ID ) ),
```

Comment:
FWIW here we only need the site IDs, so we could actually go with `Object.keys()` and use the IDs directly.

Reply:
Changing this to `Object.keys( ... )`. The difference is that in the `Object.keys( ... )` array, the IDs are strings. While `site.ID` is a number. But apparently it doesn't matter, because there are places that use `Object.keys( ... )` already.